src/shared/components/post/post-listings.tsx

```diff
@@ -115,7 +115,7 @@ export class PostListings extends Component<PostListingsProps, any> {
 
   removeDuplicates(): PostView[] {
     // Must use a spread to clone the props, because splice will fail below otherwise.
-    const posts = [...this.props.posts];
+    const posts = [...this.props.posts].filter((empty) => empty);
```

Comment:
https://stackoverflow.com/questions/281264/remove-empty-elements-from-an-array-in-javascript

Seems like `.filter(Boolean)` is a bit simpler.